regression_tests/stages/clone_demodjango_deploy.sh

```diff
@@ -2,17 +2,9 @@
 
 set -e
 
-echo -e "\nGet CodeStar connection details"
-codestar_connections=$(aws codestar-connections list-connections --provider-type GitHub --query "Connections[? ConnectionStatus == 'AVAILABLE']")
-aws_account=$(echo "$codestar_connections" | jq -r ".[0].OwnerAccountId")
-codestar_arn=$(echo "$codestar_connections" | jq -r ".[0].ConnectionArn")
-codestar_connection_id=$(echo "${codestar_arn##*/}")
-
 echo -e "\nClone demodjango_deploy"
-git config --global credential.helper '!aws codecommit credential-helper $@'
-git config --global credential.UseHttpPath true
-git clone "https://codestar-connections.eu-west-2.amazonaws.com/git-http/$aws_account/eu-west-2/$codestar_connection_id/uktrade/demodjango-deploy.git"
 
-cd demodjango-deploy
+git clone "${GIT_CLONE_BASE_URL}demodjango-deploy.git"
```

Comment:
where does `GIT_CLONE_BASE_URL` come from?

all our templated folders don't include the trailing `/` but this one does, should we keep it consistent everywhere or does it not matter? I ask because I normally see `{TEMPLATED_VAR}/the-rest-of-whatever.extension` rather than the templated var including the `/`

Reply:
Fair point will update 👍🏻

Reply:
Updated and reran the pipeline to ensure no breaking changes were introduced